tests/test_data.py

```diff
@@ -71,6 +71,43 @@ def test_proc_eye_day(self):
         self.assertIsNone(best_id)
         self.assertCountEqual(te_ids, [6581, 6577])
 
+class LoadPreprocTests_drive(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.id = '0000'
+        cls.subject = 'test'
+        cls.date = '2024-11-12'   
+        
+    def test_load_preproc_lfp_data(self):
+        lfp_data, lfp_metadata = load_preproc_lfp_data(data_dir, self.subject, self.id, self.date, drive_number=1)
+        self.assertIsInstance(lfp_data, np.ndarray)
+        self.assertIsInstance(lfp_metadata, dict)
+        lfp_data, lfp_metadata = load_preproc_lfp_data(data_dir, self.subject, self.id, self.date, drive_number=2)        
+        self.assertIsInstance(lfp_data, np.ndarray)
+        self.assertIsInstance(lfp_metadata, dict)
```

Comment:
make sure to add a test with drive_number=None for each of these

Reply:
oh nevermind it's elsewhere. it's slightly confusing to break the tests for the `load_preproc_*_data` functions into two places.. could you merge them into a single class if possible?